src/index.js

```diff
@@ -0,0 +1,195 @@
+"use strict";
+
+const MAX_DEGREE = 60;
+const MIN_DEGREE = -90;
+
+const state ={
+  //HTML states
+  increaseTempControl: null,
+  tempValue: null,
+  decreaseTempControl:null,
+  landscape: null,
+  headerCityName: null,
+  cityNameInput: null,
+  cityNameReset: null,
+  currentTempButton: null,
+  skySelect: null,
+  sky__section: null,
+  sky: null,
+  gardenContent: null,
+
+  tempValueCount: 0,
+};
+
+const loadControls = () => {
+  state.increaseTempControl = document.getElementById("increaseTempControl");
+  state.tempValue = document.getElementById("tempValue");
+  state.decreaseTempControl = document.getElementById("decreaseTempControl");
+  state.landscape = document.getElementById("landscape");
+  state.headerCityName = document.getElementById("headerCityName");
+  state.cityNameInput = document.getElementById("cityNameInput");
+  state.cityNameReset = document.getElementById("cityNameReset");
+  state.currentTempButton = document.getElementById("currentTempButton");
+  state.skySelect = document.getElementById("skySelect")
+  state.sky__section = document.getElementsByClassName("sky__section")
+  state.sky = document.getElementsByClassName("sky")
+  state.gardenContent = document.getElementById('gardenContent');
+};
+
+const getCurrentTemperature = () => {
+  const city = state.cityNameInput.value;
+
+  return axios
+    .get('http://localhost:5000/location', {params: {
```

Comment:
Consider moving the common portion of the endpoint url (http://localhost:5000) to a variable and interpolating it into the string here. That would minimize the changes required in order to point this at a deployed server (only one clearly documented variable would need to change).